docs/guides/using_ocio/tool_overview.rst

```diff
@@ -160,24 +176,44 @@ supported.
 ociodisplay
 ***********
 
-A basic image viewer. Uses OpenImageIO to load images, and displays them using
-OCIO and typical viewer controls (scene-linear exposure control and a
-post-display gamma control)
+A toy image viewer demonstrating the OCIO C++ API. 
```

Comment:
Suggest changing `toy` to `example` or `simple` for clarity.

Reply:
Agreed. "Toy" is pretty colloquial.
I wanted to make explicit that `ociodisplay` isn't (currently?) intended as full-fledged demo of all of OCIO's viewing related features (for instance, presently, it's not a reference for implementing UIs for dynamic transforms); and I'm not sure whether there are plans to update `ociodisplay` further.

If `ociodisplay` intends to serve as reference of developers, I feel like it deserves separate developer-oriented documentation (see #447), and possibly an "* ^kitchen-sink" reference config, to help DCC devs better understand how to support OCIO in practice.